tests/test_sqlalchemy/conftest.py

```diff
@@ -265,6 +265,7 @@ class UserFilter(Filter):  # type: ignore[misc, valid-type]
         name: Optional[str]
         name__neq: Optional[str]
         name__like: Optional[str]
+        name__contains: Optional[str]
```

Comment:
Here `name` is not a list of enum so I'm not sure we are testing exactly what you are looking for... Could you add another `enum` list field instead and test against it?